tests/FSharp.Compiler.ComponentTests/EmittedIL/TestFunctions/TestFunction17.fs.il.netcore.debug.bsl

```diff
@@ -124,18 +116,12 @@
                int32 V_3,
                class [runtime]System.Collections.IComparer V_4,
                int32 V_5,
```

Comment:
Also doesn't look good, unsure why baselines changed, ilx output should retain the same order